news/2017-08-25-scorewatch-august-week-3.md

```diff
@@ -0,0 +1,57 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/7TTxN6ul7ko" frameborder="0" allowfullscreen></iframe></br>
+
+[Rafis](https://osu.ppy.sh/u/2558286) continues his dominance on this article as he sets the number one score on [xi - Ascension to Heaven](https://osu.ppy.sh/b/807074) with an unfortunate 98.98% 2555x 1 miss play to snag 637pp; an FC could have been worth more than 660pp! Will this map finally be conquered soon? Maybe it might, but in the meantime, you will have to check out this insane score! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/zByevoMS2ck" frameborder="0" allowfullscreen></iframe></br>
+
+[Motion](https://osu.ppy.sh/u/3885626) has taken the number one spot in osu!catch for the first time! To celebrate this, here is the man himself setting the an impressive 99.19% NC FC on [Hitsuji to Ojisan - XENO](https://osu.ppy.sh/b/663001) to claim 1004pp! Congratulations!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/wjplwLNaIMQ" frameborder="0" allowfullscreen></iframe></br>
+
+[EtienneXC](https://osu.ppy.sh/u/5610085) tears up [Evening’s](https://osu.ppy.sh/u/2193881) [Imperial Circus Dead Decadence - FUBUKI](https://osu.ppy.sh/b/1080145) by setting an amazing 99.65% FC. Standing at 4:14 mins of drain time, this play tests a player’s control and stamina limits. With an accuracy like this, it’s not doubt Etienne is one of the craziest 4K players out there today!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/uKDiGVBDlHY" frameborder="0" allowfullscreen></iframe></br>
+
+BLAST FROM THE PAST
+-----
+
+We blast ourselves back to 2015 where [rustbell](https://osu.ppy.sh/u/227717), known for his technical ability in maps, took an amazing 99.57% HDFL FC on [IOSYS - Kanbu de Todomatte Sugu Tokeru ~ Kyouki no Udongein](https://osu.ppy.sh/b/16266) to earn 211pp. Honestly speaking, you have to see it to believe it.
+
+Once again, do support osu!archive[https://www.reddit.com/r/osugame/comments/6uracb/osuarchive_4_rustbell_iosys_kanbu_de_todomatte/] (it is still ongoing on the osugame reddit) as they approach a week of amazing old scores!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/-gtFEQQhUqI" frameborder="0" allowfullscreen></iframe></br>
+
+
+NOTABLE MENTIONS
+-----
+
+The one and only [MinG3012](https://osu.ppy.sh/u/1583218) is back as he sets a brilliant 58.12% HDHR FC on [marina - Towa yori Towa ni](https://osu.ppy.sh/b/793846?m=0) to earn 298pp. This is one crazy score.
+
+Circle size 9.1? Not a problem for [Adamqs](https://osu.ppy.sh/u/613592), securing a breathtaking 97.15% HDHR FC on [Moonbaby - Here We Go](https://osu.ppy.sh/b/371621) to earn himself 238pp. This is really frightening to watch, but what an amazing score it is!
```

Comment:
Remove 'This is' from second line to avoid double dutching

Reply:
http://www.urbandictionary.com/define.php?term=Double%20Dutching

I'm not a native speaker. Got it, though.